src/nvim/os/env.c

```diff
@@ -410,14 +410,14 @@ static char *remove_tail(char *p, char *pend, char *name)
 /// @param[out] mustfree Ouput parameter for the caller to determine if they are
 ///           responsible for releasing memory. Must be initialized to false
 ///           by the caller.
-char_u *vim_getenv(char_u *name, bool *mustfree)
+char *vim_getenv(char *name, bool *mustfree)
 {
-  char_u      *p;
-  char_u      *pend;
+  char *p;
+  char *pend;
   int vimruntime;
 
 
-  p = (char_u *)os_getenv((char *)name);
+  p = (char *)os_getenv(name);
```

Comment:
Yeah, I noted that somewhere earlier in this PR, that's not a good solution, but I was trying to limit my change to only be the u_char -> char.  I'm not adding any bad behavior, just changing it.  I have another PR to submit as soon as this one merges that refactors get_env to properly handle the const.